sdk/include/psdk/ntsecpkg.h

```diff
@@ -152,6 +154,22 @@ typedef struct _SECPKG_MUTUAL_AUTH_LEVEL {
     ULONG MutualAuthLevel;
 } SECPKG_MUTUAL_AUTH_LEVEL, *PSECPKG_MUTUAL_AUTH_LEVEL;
 
+typedef struct _SECPKG_WOW_CLIENT_DLL {
+    UNICODE_STRING WowClientDllPath;
```

Comment:
`SECURITY_STRING`?

Reply:
Do we use that type elsewhere in this file?

Reply:
No. But its defined as SECURITY_STRING in MS Headers and should be used in some other places. AFAICS SECURITY_STRING is the same as UNICODE_STRING in 32 bit mode. It differs in 64 bit.
I've added a definition to psdk/sspi.h for SECURITY_STRING.
Further i've added a commit that corrects wrong definitions in ntsecpkg.h regarding SECURITY_STRING.